app/models/draft-registration.ts

```diff
@@ -23,6 +35,21 @@ export default class DraftRegistrationModel extends OsfModel {
 
     @belongsTo('registration-schema', { inverse: null })
     registrationSchema!: DS.PromiseObject<RegistrationSchemaModel> & RegistrationSchemaModel;
+
+    @hasMany('contributor', { inverse: 'node' })
```

Comment:
```suggestion
    @hasMany('contributor', { inverse: null })
```

hmm, either that, or maybe let's omit `contributors` (and `bibliographicContributors`) for now, since we don't need contributor management for the first draft-metadata release and it's possible the api will be slightly different (what *will* the inverse relation be?)